contracts/src/v0.8/llo-feeds/v0.4.0/DestinationVerifierProxy.sol

```diff
@@ -37,14 +37,16 @@ contract DestinationVerifierProxy is IDestinationVerifierProxy, ConfirmedOwner,
     bytes calldata payload,
     bytes calldata parameterPayload
   ) external payable returns (bytes memory) {
+    if(address(s_verifier) == address(0)) revert ZeroAddress();
```

Comment:
We don't want to check this as reading and writing from contract storage is expensive. For reference:

Writing to storage where existing element is 0: 20,000 gas
Writing to storage where existing element is non 0: 5,000 gas
Reading from cold storage (1st read) : 2,000
Reading from warm storage (2nd+ read) : 100
Allocating a uint256 in memory: <30 ( memory allocation cost grows quadratically)

Given we'll always smoke test before we release the contract and a nil or bad verifier cant be set through the setter, adding this check adds additional cost to the user (2% more for verification)